docs/source/getting-started.rst

```diff
@@ -231,8 +231,8 @@ Item validation
 
 Item validators allows you to match your returned items with predetermined structure
 ensuring that all fields contains data in the expected format. Spidermon allows
-you to choose between schematics_ or `JSON Schema`_ to define the structure
-of your item.
+you to choose from schematics_,`JSON Schema`_ or `cerberus`_ to define structure and
```

Comment:
:lipstick: 
```suggestion
you to choose from schematics_,`JSON Schema`_ or cerberus_ to define structure and
```

This is something you can apply in a few more places below.

Reply:
Roger that, fixed more instance like this.